osm2streets-js/src/lib.rs

```diff
@@ -60,4 +63,41 @@ impl JsStreetNetwork {
         let road_network: streets::RoadNetwork = self.inner.clone().into();
         road_network.to_dot()
     }
+
+    #[wasm_bindgen(js_name = getDebugSteps)]
+    pub fn get_debug_steps(&self) -> Vec<JsValue> {
+        // TODO Figure out how to borrow from the RefCell instead of cloning
```

Comment:
The perf of all of this is pretty clumsy... cloning everywhere. I need to read up on lifetimes and how borrowing works with WASM. For our small test areas, it doesn't matter at all, even when compiling in debug mode though